selenium_tests/cross_listing/cross_listing_base_test_case.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from urlparse import urljoin
+
+from selenium_common.base_test_case import BaseSeleniumTestCase
+from selenium_common.pin.page_objects.pin_login_page_object \
+    import PinLoginPageObject
+
+from selenium_tests.account_admin.page_objects.account_admin_dashboard_page_object \
+    import AccountAdminDashboardPage
+from selenium_tests.cross_listing.page_objects.cross_listing_index_page \
+    import IndexPagePageObject
+
+
+class CrossListingBaseTestCase(BaseSeleniumTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(CrossListingBaseTestCase, cls).setUpClass()
+
+        cls.USERNAME = settings.SELENIUM_CONFIG.get('selenium_username')
+        cls.PASSWORD = settings.SELENIUM_CONFIG.get('selenium_password')
+
+        cls.CANVAS_BASE_URL = settings.SELENIUM_CONFIG.get('canvas_base_url')
+        cls.test_settings = settings.SELENIUM_CONFIG['cross_listing']
```

Comment:
there's a mix of using get('key') with ['key'], I think it's better to use ['key'] as the script will fail immediately if the values are not present, whereas using get will always return 'None' even if no values are present and could to a misdiagnosed failed test